Dockerfile

```diff
@@ -1,6 +1,10 @@
 # syntax=docker/dockerfile:1
 # Dockerfile is currently only for x86_64
 
+# ```sh
+# docker build ray:2.5.1-julia1.9.3-rayjl$(git rev-parse --short HEAD) .
+# ```
```

Comment:
not sure if you want to remove these lines first

Reply:
I intended to add that as a example of how someone may want to tag the package. However, now that you brought this up I see I forgot to add the `-t` flag. I'll add that and a short explanation.